components/console-backend-service/internal/domain/k8s/resourcequota_resolver.go

```diff
@@ -36,5 +43,19 @@ func (r *resourceQuotaResolver) ResourceQuotasQuery(ctx context.Context, namespa
 		return nil, gqlerror.New(err, pretty.ResourceQuotas, gqlerror.WithNamespace(namespace))
 	}
 
-	return r.converter.ToGQLs(items), nil
+	converted := r.converter.ToGQLs(items)
```

Comment:
If you don't have as second returning value error, please use `return r.converter.ToGQLs(items), nil`